packages/office-ui-fabric-react/src/utilities/positioning/positioning.types.ts

```diff
@@ -1,5 +1,4 @@
 import { DirectionalHint } from '../../common/DirectionalHint';
-import { IPoint } from './positioning.types';
```

Comment:
This line is conflicting with this export https://github.com/OfficeDev/office-ui-fabric-react/pull/10972/files#diff-7bc6de4d67a6feec646ff9ce256a7f67R103